dali/python/nvidia/dali/fn.py

```diff
@@ -88,10 +89,16 @@ def to_scalar(scalar):
     op_wrapper.__doc__ = "see :class:`nvidia.dali.ops.{0}`".format(op_class.__name__)
     return op_wrapper
 
-def _wrap_op(op_class):
+def _wrap_op(op_class, submodule):
     wrapper_name = _to_snake_case(op_class.__name__)
-    if not hasattr(sys.modules[__name__], wrapper_name):
-        setattr(sys.modules[__name__], wrapper_name, _wrap_op_fn(op_class, wrapper_name))
+    fn_module = sys.modules[__name__]
+    module = _internal.get_submodule(fn_module, submodule)
+    if not hasattr(module, wrapper_name):
+        wrap_func = _wrap_op_fn(op_class, wrapper_name)
+        setattr(module, wrapper_name, wrap_func)
+        if submodule:
+            setattr(fn_module, '.'.join(submodule + [wrapper_name]), wrap_func)
```

Comment:
I don't think this is necessary - but this is:
```suggestion
            wrap_func.__module__ = module.__name__
```